### PR TITLE
Update Hungarian translations

### PR DESCRIPTION
**Pull request by debyos**

### CODE REVIEW

**Comment by mapsmetest** [comment]
Can one of the admins verify this patch?

**Comment by debyos** [comment]
Done.

Gábor

mpimenov <notifications@github.com> ezt írta (időpont: 2019. febr. 7., Cs,
23:24):

File: data/categories.txt [hunk 1]
```diff
@@ -1495,7 +1495,7 @@ nl:^Computerwinkel
 fi:^Tietokonekauppa
 fr:^Magasin d'informatique
 de:^Computergeschäft
-hu:^Számítógép üzlet
+hu:^Számítógépüzlet
```
Comment (mpimenov) [inline data/categories.txt]: Are you sure that we don't need the version with a space?
We currently do not insert spaces automatically.

File: data/categories.txt [hunk 2]
```diff
@@ -1911,7 +1911,7 @@ nl:3^Metro
 fi:3^Metro
 fr:3^Métro
 de:2^U-Bahn|Metro|ÖPNV
-hu:3^Földalatti|3metro
+hu:3^Metró|Földalatti|3metro
```
Comment (mpimenov) [inline data/categories.txt]: Fill free to drop the last one: diacritics are ignored when looking for synonyms.

File: data/categories.txt [hunk 3]
```diff
@@ -1962,7 +1962,7 @@ nl:3^Veerpont|veerboot|veer|pont|overzetboot
 fi:3^Lautta|terminaali|kuljetus
 fr:3^Traversier|terminal
 de:2^Fähre|Terminal
-hu:2^Komp
+hu:2^Kompkikötő
```
Comment (mpimenov) [inline data/categories.txt]: Is it really so rare (or important) to start suggesting it after typing "Ko" (2 characters)?

File: data/categories.txt [hunk 4]
```diff
@@ -2259,7 +2259,7 @@ nl:^Kerk|tempel|kathedraal|gebedsplaats|gebedshuis
 fi:^Kirkko|uskonnollinen kohde
 fr:^Église|lieu de culte|temple|basilique
 de:^Kirche|Dom|3Kathedrale|4Münster|Anbetungsstätte|Tempel
-hu:^Templom
+hu:^Templom|Székesegyház|Katedrális|Dóm|Bazilika|Istentisztelei hely
```
Comment (mpimenov) [inline data/categories.txt]: Should the last one be "Istentiszteleti hely"?

File: data/categories.txt [hunk 5]
```diff
@@ -2831,7 +2831,7 @@ el:^Παραχώρηση αυτοκινήτου
 es:^Compartir coche
 fi:^Autojen yhteiskäyttö
 fr:^Car Sharing
-hu:^Gépjármű megosztása
+hu:^Gépjárműmegosztás
```
Comment (mpimenov) [inline data/categories.txt]: Same here: should we leave ```Gépjárműmegosztás|Gépjármű megosztás```?

File: data/categories.txt [hunk 6]
```diff
@@ -3797,7 +3797,7 @@ nl:3^Toilet
 fi:^WC
 fr:3^Toilettes
 de:3^Toilette
-hu:3^Mosdó
+hu:3^WC
```
Comment (mpimenov) [inline data/categories.txt]: Please remove the "3".

File: data/categories.txt [hunk 7]
```diff
@@ -5378,7 +5378,7 @@ nl:3^Auto reparatie|auto|wagen
 fi:^Autokorjaamo
 fr:3^Réparation d'automobiles|3garage|4station service|voiture|auto
 de:3^Autowerkstatt|Kfz|Reparaturwerkstatt|Auto|Werkstatt
-hu:3^Gépkocsi szervíz|4javítóműhely|kocsi
+hu:3^Autószerelő|Autószerviz|Gépkocsiszerviz|Autójavító|4javítóműhely|kocsi
```
Comment (mpimenov) [inline data/categories.txt]: Should we really remove the old synonym?

File: data/categories.txt [hunk 8]
```diff
@@ -9228,7 +9228,7 @@ el:^Καλύβα|αγροικία
 es:^Cabaña
 fi:^Autiotupa
 fr:^Cabane sauvage
-hu:^Kunyhó a vadonban
+hu:^Menedékház (személyzet nélkül)
```
Comment (mpimenov) [inline data/categories.txt]: Note that this file contains synonyms for search: we will suggest the category if a user types in one of the strings separated by "|". Here someone needs to type in (almost exactly) ```Menedékház (személyzet nélkül)``` which I think is unlikely. Please try to find a shorted synonym: what would you type in the search bar if you wanted to find a wilderness hut?

**Comment by mpimenov** [comment]
You may also be interested in taking a look at data/strings/strings.txt: https://github.com/mapsme/omim/blob/master/data/strings/strings.txt

File: data/categories.txt (hunk 1 above)
Comment (debyos) [reply data/categories.txt]: Yes, I am sure.
In Hungarian, if you have a composed noun which is made of two parts or more than two parts but is at most 6 sillables long, it must be written together.

File: data/categories.txt (hunk 2 above)
Comment (debyos) [reply data/categories.txt]: OK, I will drop it (or you can do as well).

File: data/categories.txt (hunk 3 above)
Comment (debyos) [reply data/categories.txt]: No, sorry, I just took over from the previous version.

File: data/categories.txt (hunk 4 above)
Comment (debyos) [reply data/categories.txt]: Yes, you're right. Thanks for spotting!

File: data/categories.txt (hunk 5 above)
Comment (debyos) [reply data/categories.txt]: The same rule here like above (this is the so-called 6:3 rule: 6 sillables and 3 parts).

File: data/categories.txt (hunk 6 above)
Comment (debyos) [reply data/categories.txt]: OK, I will do it.

File: data/categories.txt (hunk 7 above)
Comment (debyos) [reply data/categories.txt]: It is there but written correctly: _Gépkocsiszerviz_ (the third expression, again, written without spaces)

File: data/categories.txt (hunk 8 above)
Comment (debyos) [reply data/categories.txt]: It can be just _menedékház_ ("személyzet nélkül" means "without personnel" which is not indispensable)

**Pull request merged**